httputil/httputil.go

```diff
@@ -213,10 +215,16 @@ func DownloadBinary(originURL, destDir, destFile string, config config.Config) (
 		log.Printf("Downloading %s...", originURL)
 
 		var auth string = ""
-		t, err := tryFindNetrcFileCreds(u.Host)
-		if err == nil {
-			// successfully parsed netrc for given host
-			auth = t
+		if AuthHeader != "" {
```

Comment:
I think we should raise an error if both AuthHeader is set and a .netrc file exists.